src/components/Error/Empty.jsx

```
@@ -27,17 +28,20 @@ const EmptyCanvas = ({
   const { t } = useI18n()
   const { isDesktop } = useBreakpoints()
   const { displayedFolder } = useDisplayedFolder()
+  const isSharedDriveEnabled = flag('drive.shared-drive.enabled')
 
   const IconToShow = type === 'trash' ? TrashIllustration : FolderEmptyIllu
   const showUploadLayout =
-    (type === 'drive' || type === 'encrypted') && canUpload
-  const showSharedDriveLayout = type === 'sharing'
+    type === 'drive' ||
+    (type === 'encrypted' && canUpload) ||
+    showSharedDriveLayout
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)